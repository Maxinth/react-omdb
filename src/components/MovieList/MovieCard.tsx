import { Card } from "./movieList.styled";
import { ListItem } from ".";
import React from "react";

const MovieCard = (props: ListItem) => {
  const { Title } = props;
  return (
    <Card
    //  style={{ backgroundImage: `url(${Poster})` }}
    >
      {Title}
    </Card>
  );
};

export default MovieCard;
