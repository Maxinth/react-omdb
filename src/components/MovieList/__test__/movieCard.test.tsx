import MovieCard from "../MovieCard";
import renderer from "react-test-renderer";
import React from "react";

const mockProps = {
  Poster: "me",
  Title: "snapshots",
  Type: "first-type",
  Year: "2022",
  imdbID: "---",
};

it("renders a snapshot of <MovieCard />", () => {
  const card = renderer.create(<MovieCard {...mockProps} />).toJSON();
  expect(card).toMatchSnapshot();
});
