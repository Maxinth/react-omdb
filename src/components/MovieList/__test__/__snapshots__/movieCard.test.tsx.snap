// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <MovieCard /> 1`] = `
<div
  className="sc-eCImPb iTldMi"
>
  snapshots
</div>
`;
